### ISSUE TITLE
Deactivate "trailingSlash" logic

### ISSUE THREAD

**Issue by fdc-viktor-luft** [vercel/next.js#39718]
### Describe the feature you'd like to request

Currently (according to [this](https://nextjs.org/docs/api-reference/next.config.js/trailing-slash)) it is only possible to decide wether NextJS should or should not use trailing slashes. Using a middleware on top of that doing further redirects can result into more than a single redirect, which is bad in terms of SEO. It would be nice if these internal trailing-slash redirects could be deactivated and handled manually in the middleware to allow single redirects. 

### Describe the solution you'd like

```js
module.exports = {
  trailingSlash: null, // no trailing slash logic
}
```

### Describe alternatives you've considered

We've considered forwarding sanitized URLs from Virtual-Service to the NextJS application, but ran into issues because NextJS server didn't like this mismatch and wouldn't allow to send the correct headers for a redirect back. We seem to have no control over these internal redirects. Allowing us to deactivate these, gives us more control over the incoming requests.

**Comment by SukkaW** [comment]
> ...doing further redirects can result into more than a single redirect, which is bad in terms of SEO

That is not true. For Google, Bing, and Yandex, as long as multiple redirects all have the desired status code (`301` and `308` for permanent).

https://developers.google.com/search/docs/advanced/crawling/301-redirects
